dev-python/python-language-server/python-language-server-0.31.2.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit eutils distutils-r1
+
+DESCRIPTION="An implementation of the Language Server Protocol for Python"
+HOMEPAGE="https://github.com/palantir/python-language-server"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/python-jsonrpc-server[${PYTHON_USEDEP}]
+		dev-python/jedi[${PYTHON_USEDEP}]
+		dev-python/pluggy[${PYTHON_USEDEP}]
+		dev-python/ujson[${PYTHON_USEDEP}]"
```

Comment:
Please sort alphabetically and confirm these are not needed during runtime but during build-time.

I see upstream has some tests packaged, can you try enabling them?